### DISCUSSION TITLE
Strange csv report "attributes" column type

### DISCUSSION THREAD

**Discussion by lombardata** [biigle/core#447] [Q&A] [answered]
Hi,
Today i faced a problem that i had never encountered before in Biigle and wanted to know if anyone had an idea how to solve it.
I've annotated a volume of images and requested a csv report but when I open it, in the "attributes" column instead of having a json object as was the case with all the other volumes of images I have annotated and as explained in the user manual :
**The additional attributes of the image are encoded as a JSON object. The content may vary depending on the BIIGLE modules that are installed and the operations performed on the image (e.g. a laser point detection to calculate the area of an image).**
I've a 'float' object. 
Moreover, I don't know if there is a relation between these two events, when I open the project page where all the volumes that are part of the project are shown, the volume in question is the only one that does not present a preview of the images contained in it.
Could someone help me?
Thanks,
Matteo
![Screenshot from 2022-04-21 11-15-54](https://user-images.githubusercontent.com/39915110/164396525-a6510e16-bbc4-435b-9da6-d19faef83e1c.png)

**Comment by mzur** [comment, marked as answer]
Hi Matteo, in your CSV the `attributes` column seems to be empty (as the values in `points` overflow to this column). Maybe your script interprets empty values as float. I assume that BIIGLE was unable to process the images when the volume was created. This way, it could not generate preview thumbnails and it could not collect image metadata which is normally stored in `attributes`.

I've also opened a new issue (https://github.com/biigle/reports/issues/87) to fix this behavior. If no image attributes are present, the CSV report should contain an empty JSON object for the image instead of no value.

**Reply by lombardata** [reply]
Ok, thank you very much. In order to temporarily fix the problem, I'll manually add an empty json object.
Do you know why Biigle was unable to process the images when the volume was created?
Please find attached an example of  image of the volume in question,
Have a good day,
Matteo 
![session_2020_10_10_Europa_image_DSCI1671](https://user-images.githubusercontent.com/39915110/164430061-e6769c23-1d4b-497e-8801-f809371f3bda.JPG)

**Reply by mzur** [reply]
BIIGLE writes a warning message to the log file in this case. It should say something like: "Could not process new image [id]: [error message]." You can find the log messages in the BIIGLE admin area in the "Logs" section or as a file in `storage/logs/`.

**Reply by lombardata** [reply]
Ok, thank you very much !

**Reply by mzur** [reply]
Please click "Mark as answer" if your issue was resolved. Otherwise, please don't hesitate to continue the discussion :wink:
